css/styles.css

```diff
@@ -0,0 +1,78 @@
+/*
+- Font sizes (px):
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+
+- Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+*/
+
+:root {
+  font-size: 10px;
+  font: 400 10px / 1.2 "Yeseva One";
```

Comment:
Looks like `10px` is set double times

Reply:
Because in my browser the default font-size is 20px and it's weird. Chrome: Version 116.0.5845.110

Reply:
I use Brave (Version 1.61.104 Chromium: 120.0.6099.115), it works well. Also never had issues with that.